recipes/python-louvain/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "python-louvain" %}
+{% set version = "0.10" %}
+{% set sha256 = "3b50d536f3eaf7363ae3c4167916ec7b055a4c70efc7c379b8e4c8dec724f6b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Can this package be noarch? Looks like it is possible but I did not see the code to check if there are compiled parts or some automatic py2/3 conversion in `setup.py`.

Reply:
Looks like so. There's no Cython, no 2/3 conversions, so I'm going to make this happen then.